Secrets of Azeroth

wyldclaw opened the pull request:
Add Secrets of Azeroth event nodes

Commit:
Day 1 initial commit

I know localization is in the wrong place. It's a work in progress!

Commit:
Add second day

Commit:
Add Secrets of Azeroth localization

Commit:
Add GetCoinTextureString global

Commit:
Update Day 1 and finish Day 2

Krovikan committed:
Updated Secrets of Azeroth esES

Lightuky committed:
Update frFR.lua - Secrets of Azeroth - DF

Commit:
Fix lua error for GetCoinTextureString()

Dathwada committed:
Update deDE.lua, Fix typo in enUS.lua

Dathwada committed:
Update Secrets Of Azeroth

* moved localizations from common.lua
* FangliClue icons appear when the quest Unfinished Thinking Cap is completed
* Found the Hungering Tyranha in Valdrakken
* Some icons will now disappear if the player has the item (or the crafted item) in his bags
* The quests 78202 and 77513 seem to be accountwide

Dathwada committed:
Fix lint error

Lightuky committed:
Update frFR.lua - Secrets of Azeroth - DF

Commit:
Update NPC ID for Crazed Looter

Commit:
Merge branch 'secrets-of-azeroth' of https://github.com/zarillion/handynotes-plugins into secrets-of-azeroth

Krovikan committed:
Updated gryffin_note_b + options SoA esES

Lightuky committed:
Update frFR.lua - soa_day02_gryffin_note_b - DF

Dathwada committed:
Update Secrets Of Azeroth (day 1)

* add requires to made it more clear what you need
* add questDeps and playerHasItem to show / hide icons
* remove rewards for Golden Chalice because you don't really get a reward from this step you just place the chalice there

Krovikan committed:
Updated emote esES

Commit:
Add first Buried Satchel

Commit:
Merge branch 'secrets-of-azeroth' of https://github.com/zarillion/handynotes-plugins into secrets-of-azeroth

Krovikan committed:
Updated plague_plaque esES

adavak committed:
Update zhCN.lua

adavak committed:
Update enUS.lua

adavak committed:
Update enUS.lua

Commit:
Update Buried Satchel and add 2 additional finds

Commit:
Add Day 3

Krovikan committed:
Updated Day 3 + Buried Satchel + fixes esES

Krovikan committed:
Forgotten blank line enUS

Lightuky committed:
Update frFR.lua - day 3 notes - DF

Dathwada committed:
Update deDE.lua

Commit:
Add 4th Buried Satchel

Commit:
Update enUS.lua

Commit:
Update secrets_of_azeroth.lua

Krovikan committed:
Updated bs_smv_location esES

adavak committed:
Update zhCN.lua

adavak committed:
fix.

Commit:
Add Day 4

Krovikan committed:
Updated Day 4 esES

Lightuky committed:
Update frFR.lua - day 4 + fix - DF

Dathwada committed:
Update deDE.lua

Dathwada committed:
Update secrets_of_azeroth.lua

Lightuky committed:
Update frFR.lua - fix note - DF

Commit:
Add 5th satchel

Commit:
Merge branch 'secrets-of-azeroth' of https://github.com/zarillion/handynotes-plugins into secrets-of-azeroth

Commit:
Fix note

Lightuky committed:
Updade frFR.lua - bs_net_location - DF

Krovikan committed:
Updated 5th Satchel + fix esES

Ioney committed:
fixed typos in deDE, added POI for Odd Statue

adavak committed:
zhCN.

Commit:
Massive update/rewrite

Commit:
Innocuous change to try lint again

Lightuky committed:
Update frFR.lua - Clue 6 - DF

adavak committed:
zhCN.

Ioney committed:
deDE SoA rLabel updated

Krovikan committed:
Updated Clue 6 + fix esES

Lightuky committed:
Update frFR.lua - fix typo - DF

Krovikan committed:
Fixed esES

Dathwada committed:
Update deDE.lua

Dathwada committed:
Update Secrets of Azeroth

All these nodes were still shown even though I had completed them.

* 77230 the bar tab quest will reset to false once you got the Crystal Ocular Lenses from Shakey Flatlap
* Added quest ids for the Torch of Pyrreth POIs and PATHs, this will hide them after the player activated the lever
* Added Bobby Carlisle for Clue 36

Commit:
Add Buried Satchel super node for tracking

Krovikan committed:
Updated soa_06_unvieled_tablet_note_a + fix esES

Krovikan committed:
More fixes esES

adavak committed:
Update zhCN.lua

Commit:
Add new Buried Satchel. Fix checklist node.

Commit:
Update zhCN.lua

adavak committed:
zhCN.

Commit:
Add clue 7

zarillion commented:
Let me know when this is ready to merge and I will get a quick release out.

adavak committed:
Update zhCN.lua

adavak commented:
> Let me know when this is ready to merge and I will get a quick release out.

Clue 15, 9 to go.

Lightuky committed:
Update frFR.lua - clue 7 + satchel - DF

Krovikan committed:
Updated Satchel + Clue 7 esES

Dathwada committed:
Update Secrets of Azeroth Clue 07

* Add start quest step
* Fix quest ids for Brazier 2, Brazier 3 and Using the Idol

Dathwada committed:
Update deDE.lua

Lightuky committed:
Update frFR.lua - fix clue 7 - DF

Krovikan committed:
Updated intro quest + fix esES

Commit:
Add 7th buried satchel

Commit:
Add clue 8

Lightuky committed:
Update frFR.lua - clue 8 + satchel - DF

Krovikan committed:
Updated Satchel 7 + Clue 8 esES

adavak committed:
Update zhCN.lua

Krovikan committed:
Fixed itemID for Clue 8 esES

Krovikan committed:
Fixed article esES

Commit:
Fix satchel count localization

Lightuky committed:
Update frFR.lua - fix item - DF

Commit:
Centralize buried satchel data

Dathwada committed:
Update deDE

Ioney committed:
Updated Clue 7 Quests, added Kathos to Clue 8

Added all 3 quests to the Brazier class,  the Quests you complete at the Braziers are not specific, you can even complete all 3 at the same Brazier.

Added Kathos node for Clue 8. (altho he did not gave the quest to me to start Clue 8.

Ioney committed:
deDE buried_satchel_sublabel update

Commit:
Add Buried Satchel 8

Krovikan committed:
Updated Kathos for Clue 8 esES

Ioney committed:
formatting fix

Krovikan committed:
Updated Satchel 8 esES

Commit:
Start Mimiron's Jumpjets

Lightuky committed:
Update frFR.lua - fix clue 8 + satchel - DF

adavak committed:
zhCN.

adavak committed:
Update secrets_of_azeroth.lua

Dathwada committed:
Update Secrets of Azeroth

* (Clue 08) 76509 is the quest [Into the Sands](https://www.wowhead.com/quest=76509) but if you complete it this quest 77854 changes to true
* (Clue 02) Add back quest id for Elder Poa
* Update deDE.lua
* Fix typo in enUS.lua

Dathwada committed:
Add SoA Clue 09

* Feel free to improve the enUS localization.

Krovikan committed:
Updated Clue 9 + fixes esES

Ioney committed:
Mimiron's Jumpjets - Part 1

The clues leading to the Braziers in Stranglethorn are not needed to get the Booster Part.

Lightuky committed:
Update frFR.lua - Mimiron's Jumpjets + clue 9 - DF

Ioney committed:
MJJ List added to Valdrakken

Dathwada committed:
Add Buried Satchel 9

Dathwada committed:
Fix parentMapID for First Booster Part

* see my last commit 322a286b

Krovikan committed:
Updated Satchel 9 + Mimiron esES

Krovikan committed:
Fixed esES

Krovikan committed:
Another fix esES

adavak committed:
zhCN.

Krovikan committed:
Dot fixed esES

Lightuky committed:
Update frFR.lua - fix mimiron + satchel - DF

Ioney committed:
MJJ Part 2+3, Satchel 10

Lightuky committed:
Update enUS.lua - fix typo + add spell ID - DF

Lightuky committed:
Update frFR.lua - MJJ Part 2 & 3 + Satchel - DF

Lightuky committed:
Update enUS.lua - npc ID + object name - DF

Lightuky committed:
Update frFR.lua - npc ID + object name - DF

Lightuky committed:
Update enUS.lua/frFR.lua - fix note - DF

Krovikan committed:
Updated Mimiron JumpJets 2 & 3 esES

Ioney committed:
update deDE - MJJ Part2

Ioney committed:
added Clue 10

Ioney committed:
Update enUS.lua

mcc1 committed:
update zhTW.lua

Ioney committed:
Clue 11 (wip)

Ioney committed:
soa_11_rlabel lint fix

Ioney committed:
zhTW lint fix

oops

Lightuky committed:
Update frFR.lua - clue 10 + 11 - DF

Krovikan committed:
Updated Clue 10 & 11 esES

Dathwada committed:
Update SoA Clue 10

* Update deDE
* For the Rose Gold Dust the quest 70513 never triggered for me and the icon was still showing after completing clue 10.

mcc1 committed:
update zhTW.lua

mcc1 committed:
update zhTW.lua

mcc1 committed:
update zhTW.lua

Ioney committed:
MJJ Part4 + Notes for Clue 11

mcc1 committed:
update zhTW.lua

Krovikan committed:
Added tag to Legion Cannons enUS

Krovikan committed:
Updated Mimiron + fixes esES

Krovikan committed:
Little fix esES

Lightuky committed:
Update frFR.lua - mjj_part4 + fix - DF

Ioney committed:
Update Clue 11 + Optional steps

Krovikan committed:
Updated Clue 11 esES

Krovikan committed:
Added tag for Clue 11 enUS

Lightuky committed:
Update frFR.lua - fix clue 11 - DF

Dathwada committed:
Update SoA Clue 11

* 77854 belongs to Clue 08s quest Into the Sands (76509)
* Fix Jhara id
* Fix Fifth Marker coords
* Fix Burial Banner lua error with pois
* playerHasItem works the other way around, it hides the nodes if the player has the item, replaced with requires

Ioney committed:
small fixes for clue 11 + started clue 12

Ioney committed:
Clue 12

Commit:
Update enUS.lua

Commit:
Update enUS.lua

Commit:
Fix note for 5th clue

Commit:
Fix Jumpjets and lua errors

Lightuky committed:
Update frFR.lua - clue 12 + fixes - DF

Krovikan committed:
Updated Clue 12 + fixes esES

adavak committed:
zhCN.

Commit:
Add 11, 12, and 13 buried satchels

adavak committed:
Update zhCN.lua

Commit:
Add clue 13?

Commit:
Merge branch 'secrets-of-azeroth' of https://github.com/zarillion/handynotes-plugins into secrets-of-azeroth

Commit:
Add 14 satchel

Commit:
Add final satchel. Hide super node when all satchels are found.

Commit:
Update secrets_of_azeroth.lua

Krovikan committed:
Fixed typo enUS

Krovikan committed:
Updated Clue 13 + Satchels 11 - 15 esES

Krovikan committed:
Forgotten line Clue 13 esES

Krovikan committed:
Fixed typo esES

Dathwada committed:
Update SoA clue 13 and satchels

Krovikan committed:
Updated Clue 13 esES

Ioney committed:
Satchel 16 (Officialy 15)

Lightuky committed:
Update frFR.lua - clue 13 + satchels - DF

Krovikan committed:
Updated Satchel 16 esES

Commit:
Swap satchel order

Commit:
Add buried satchels to parent map

Commit:
Add mount check for jump part nodes

Commit:
Ugh this lint is so aggressive

Commit:
Temporary push I know it has errors

Commit:
Add clue 14, fix jumpjet node

Commit:
Oops! Add back some properties

Lightuky committed:
Update frFR.lua - clue 14 - DF

Krovikan committed:
Updated Clue 14 esES

Dathwada committed:
Update SoA Clue 14

Dathwada committed:
Add Satchel 17

Commit:
Add 17th satchel

Commit:
Merge branch 'secrets-of-azeroth' of https://github.com/zarillion/handynotes-plugins into secrets-of-azeroth

Commit:
Minor update to satchel 17 and satchel tracking

Lightuky committed:
Update frFR.lua - satchel 17 - DF

Krovikan committed:
Updated Satchel 17 esES

Commit:
Add final clue 15

Lightuky committed:
Update frFR.lua - clue 15 - DF

adavak committed:
zhCN.

Krovikan committed:
Updated Clue 15 esES

Dathwada committed:
Update SoA Clue 15

Dathwada committed:
Update deDE.lua

mcc1 committed:
update zhTW.lua

mcc1 committed:
update zhTW.lua

Krovikan committed:
Some fixes esES

Commit:
Fix booster part visibility

Commit:
Update booster part again

Ioney committed:
Clue 11 optional steps fix

Commit:
Final booster part change?
